infra/cifuzz/platform_config/prow.py

```diff
@@ -12,11 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Module for getting the configuration CIFuzz needs to run on prow."""
+import logging
+import os
 import platform_config
 
 # pylint: disable=too-few-public-methods
 
 
 class PlatformConfig(platform_config.BasePlatformConfig):
   """CI environment for Prow."""
-  # TODO(mpherman2) Fill this out.
+
+  @property
+  def project_src_path(self):
+    """Returns the manually checked out path of the project's source if
+    specified or the current directory if not. Prow will run ClusterfuzzLite
+    at the directory head for the repo."""
+    project_src_path = os.getenv('PROJECT_SRC_PATH', '.')
+    logging.debug('PROJECT_SRC_PATH: %s.', project_src_path)
+    return project_src_path
+
+  @property
+  def workspace(self):
+    """Returns the workspace."""
+    # Let Prow user override workspace, but default to using artifacts dir
+    return os.getenv('WORKSPACE', os.getenv('ARTIFACTS'))
```

Comment:
This seems a bit weird to me. Why is it the ARTIFACTS directory?

Reply:
This lets users get the crash/fuzzer from the prow front page without having access to the GCB bucket. It is nice to be able to download the fuzzer and crash straight from the prowjobs artifacts page.

Reply:
> I get that Prow users might not want to use the gsutil filestore. Theoretically they could want to use no filestore, or another filestore that is implemented in the future. It would be nice though to set gsutil as the default `os.getenv('FILESTORE', 'gsutil')`just to reduce the default setup

Oh I definitely agree with this. Same for GCB users.

Reply:
We put a lot of stuff in the workspace besides crashes though. Also corpora and builds (as you point out) and coverage reports.
Making crashes easy to download is pretty compelling though.
Can you set the artifacts directory so that it's $WORKSPACE/out/artifacts (as is done by [CIFuzz users](https://github.com/systemd/systemd/blob/main/.github/workflows/cifuzz.yml#L48)) so only crashes are put in artifacts? I can jump on a video chat if this is unclear.
Anyway, if we don't find a better solution this is acceptable.